test/unittest/unittest_nntrainer_modelfile.cpp

```diff
@@ -358,13 +350,13 @@ GTEST_PARAMETER_TEST(
   /**< positive: basic valid scenarios (2 positive and 3 negative cases) */
      mkIniTc("basic_p", {nw_base_mse, adam, input + "-Activation", out+"input_layers=inputlayer" + "-Activation"}, SUCCESS),
      mkIniTc("basic2_p", {nw_base_mse, sgd, input + "-Activation", out+"input_layers=inputlayer" + "-Activation"}, SUCCESS),
-     mkIniTc("basic3_p", {nw_base + "loss=cross", adam, input + "-Activation", out+"input_layers=inputlayer" + "-Activation"}, SUCCESS),
-     mkIniTc("basic4_p", {nw_base + "loss=cross", adam, input + "-Activation", out+"input_layers=inputlayer" + "-Activation"}, SUCCESS),
+     mkIniTc("basic3_p", {nw_base + "loss=cross", adam, input + "-Activation", out+"input_layers=inputlayer"}, SUCCESS),
+     mkIniTc("basic4_p", {nw_base + "loss=cross", adam, input, out+"input_layers=inputlayer"}, SUCCESS),
```

Comment:
Pure question : 
Why should we remove `-Activation` from here to pass?

Reply:
In the current nntrainer, when using cross loss, either a sigmoid or softmax activation must be used. 
Therefore, if you delete the activation in out, it will not work properly. 

It's probably different from the previous implementation, so I modified it for pass.

Reply:
Got it, thanks for your explanation!